index.html

```diff
@@ -97,13 +97,13 @@ <h2 class="card__content--heading">Tonic</h2>
           height="448">
         </div>
         <div class="card__content">
-          <h2 class="card__content--heading">Multi-Post Stories</h2>
+          <h1 class="card__content--heading">Multi-Post Stories</h1>
```

Comment:
- [ ] Instead of using h1 here blush Please use h2 because this is the main heading of your page and according to the [heading hierarchy](https://www.w3.org/WAI/test-evaluate/preliminary/#headings) :+1: ,
- [ ] Kindly make sure you apply this to every heading with the class `card__content--heading` to meet the requirement :+1: